images/runtimes/go1.12/Dockerfile

```diff
@@ -1,4 +1,8 @@
 # Go is easy, since we already have everything we need to compile the proctor
 # binary and run the tests in the golang Docker image.
 FROM golang:1.12
-RUN ["go", "tool", "dist", "test", "-compile-only"]
+# This step will take a lot of time in a cross-compilation environment.
```

Comment:
I think we'll just need to bite the bullet, get the scripts working, let it compile and push? I can do that so it won't have to cross-compile.